store/sqlstore/group_store.go

```diff
@@ -1035,3 +1045,81 @@ func (s *SqlGroupStore) GetGroups(page, perPage int, opts model.GroupSearchOpts)
 
 	return groups, nil
 }
+
+func (s *SqlGroupStore) ifGroupsThenTeamUsersRemovedQuery(teamID string, groupIDs []string, isCount bool) squirrel.SelectBuilder {
+	var selectStr string
+
+	if isCount {
+		selectStr = "count(DISTINCT Users.Id)"
+	} else {
+		tmpl := "Users.*, TeamMembers.SchemeGuest, TeamMembers.SchemeAdmin, TeamMembers.SchemeUser, %s AS GroupIDs"
+		if s.DriverName() == model.DATABASE_DRIVER_MYSQL {
+			selectStr = fmt.Sprintf(tmpl, "group_concat(UserGroups.Id)")
+		} else {
+			selectStr = fmt.Sprintf(tmpl, "string_agg(UserGroups.Id, ',')")
+		}
+	}
+
+	groupIDsSQL := strings.Join(groupIDs, "', '")
```

Comment:
same problem here with the concatenation and interpolation of this string, I don't trust this kind of stuff, can we try to build it with squirrel?

Reply:
I have updated it to be as much in Squirrel as possible but I don't think there's a way to completely eliminate the `fmt.Sprintf` parts without
a) doing something hacky with squirrels args, and
b) doing a conditional on the db driver type